Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -893,81 +916,79 @@ - (instancetype)initWithParentAdapter:(ALVungleMediationAdapter *)parentAdapter
     return self;
 }
 
-- (void)bannerAdDidLoad:(VungleBanner *)banner
+- (void)bannerAdDidLoad:(VungleBannerView *)bannerView
 {
-    [self.parentAdapter log: @"AdView loaded: %@", banner.placementId];
-    
-    NSString *creativeIdentifier = banner.creativeId;
+    [self.parentAdapter log: @"AdView loaded: %@", bannerView.placementId];
+
+    NSMutableDictionary *extraInfo = [NSMutableDictionary dictionaryWithCapacity: 3];
+
+    NSString *creativeIdentifier = bannerView.creativeId;
     if ( [creativeIdentifier al_isValidString] )
     {
-        [self.delegate didLoadAdForAdView: self.parentAdapter.adViewContainer withExtraInfo: @{@"creative_id" : creativeIdentifier}];
+        extraInfo[@"creative_id"] = creativeIdentifier;
     }
-    else
+
+    // TODO: We are confirming with MAX if we need to pass bannerView's w and h through this callback or not.
+    CGSize adSize = bannerView.vungleAdSize.size;
+    if ( !CGSizeEqualToSize(CGSizeZero, adSize) )
     {
-        [self.delegate didLoadAdForAdView: self.parentAdapter.adViewContainer];
+        extraInfo[@"ad_width"] = @(adSize.width);
+        extraInfo[@"ad_height"] = @(adSize.height);
```

Comment:
If `extraInfo` is empty, do we still need to pass the  `extraInfo` parameter?

Reply:
`size` in `VungleAdSize` shouldn't be empty.

Reply:
Updated.